crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -195,6 +211,24 @@ impl GlobalTransform {
         value
     }
 
+    /// Returns the result of applying this [`GlobalTransform`] to a [`Vec3`] interpreted as a point.
+    ///
+    /// This applies rotation, scale, and translation. It's the equivalent of using w=1 in
+    /// homogeneous coordinates.
```

Comment:
Could you link an appropriate explanatory article on homogenous coordinates as part of this? It's pretty dense for beginners.

https://www.tomdalling.com/blog/modern-opengl/explaining-homogenous-coordinates-and-projective-geometry/ looks quite nice.